modules/ossm-tutorial-bookinfo-removing.adoc

```diff
@@ -10,8 +10,9 @@ When you finish with the Bookinfo application, you can remove it by running the
 
 [TIP]
 ====
-Several of the other {ProductShortName} tutorials also use the Bookinfo application.  Do not run the cleanup script if you plan to explore other {ProductShortName} tutorials.
+Several of the other {ProductShortName} tutorials also use the Bookinfo application. Do not run the cleanup script if you plan to explore other {ProductShortName} tutorials.
 ====
+
 .Procedure
 
 . Download the cleanup script:
```

Comment:
The link below is dead.  It should be sufficient to replicate the `apply` commands, using `delete` instead.  For example,

```
$ oc delete -n bookinfo -f https://raw.githubusercontent.com/Maistra/bookinfo/maistra-1.0/bookinfo-gateway.yaml
$ oc delete -n bookinfo -f -f https://raw.githubusercontent.com/Maistra/bookinfo/maistra-1.0/bookinfo.yaml
```

You'll also need to add a command to delete the destination rules, depending on which file they installed (i.e. whether using tls or not).

Reply:
The install script process is deprecated. Please see the updated file.